src/fmu/ensemble/ensemble.py

```diff
@@ -130,6 +137,10 @@ def __init__(
         if isinstance(runpathfile, pd.DataFrame) and not runpathfile.empty:
             count = self.add_from_runpathfile(runpathfile, runpathfilter)
 
+        if manifest:
```

Comment:
Since manifest is not Bool, suggest to use `if manifest is not None:`

Reply:
`if manifest` will ignore an empty dict, and other empty containers, which was kind of the intention. But the other way would work too.